Exiled.Events/EventArgs/Scp173/AddingObserverEventArgs.cs

```diff
@@ -0,0 +1,52 @@
+// -----------------------------------------------------------------------
+// <copyright file="AddingObserverEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp173
+{
+    using API.Features;
+    using Interfaces;
+
+    /// <summary>
+    /// Contains all information before a players just saw scp 173 (useful for an UTR for example).
```

Comment:
```suggestion
    /// Contains all information before a player sees scp 173.
```
Remove example and doc rework